drivers/windows/windows.go

```diff
@@ -188,7 +188,7 @@ func (d *Driver) CreateReadWrite(id, parent string, opts *graphdriver.CreateOpts
 }
 
 // Create creates a new read-only layer with the given id.
-func (d *Driver) Create(id, parent string, opts *graphdriver.CreateOpts) error {
+func (d *Driver) Create(id, parent, mountLabel string, opts *graphdriver.CreateOpts) error {
```

Comment:
Is this left over from an earlier iteration of the patch?

Reply:
Fixed